src/core/Table/TableRowMemoized.tsx

```diff
@@ -48,7 +50,14 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   return (
     <>
-      <div {...rowProps} ref={rowRef}>
+      <div
+        {...rowProps}
+        key={rowProps.key}
```

Comment:
Is it needed here as key is added in Table?

Reply:
`rowProps.key` was explicitly destructed before it was removed in #50, so I guess it was just a merge (This PR was created before #50 was merged).

Removed.